### ISSUE TITLE
Print server-client mismatch when messages can't be deserialized

### ISSUE THREAD

**Issue by bitmingw** [CorfuDB/CorfuDB#785]
In development and testing process, sometimes developers hit the following exception when the client want to connect to corfu server.

```
org.corfudb.runtime.clients.NettyClientRouter$2event-0 NettyClientRouter - Exception during channel handling.
io.netty.handler.codec.DecoderException: java.lang.RuntimeException: Attempt to deserialize a message which is not a CorfuMsg, Marker = 0 but expected 0xC0FC0FC0
                at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:442) ~[netty-all-4.1.6.Final.jar:4.1.6.Final]
```

Most of time this exception happens when the version of server is not matched to the version of client. Is there a way to put a prompt message in `CorfuMsg` to help people troubleshooting this issue?

**Comment by bitmingw** [comment]
Not seen this problem for a long time, thus not necessary to introduce this feature for debugging.

**Comment by no2chem** [comment]
@bitmingw This should also be addressed somewhat by #879
